docker-compose/datasets/Dockerfile

```diff
@@ -27,6 +27,6 @@ ADD shanoir-ng-datasets.jar shanoir-ng-datasets.jar
 COPY entrypoint entrypoint_common oneshot /bin/
 
 # Use the below line for remote debugging and to active dev profile
-#ENTRYPOINT ["/bin/entrypoint", "java", "-Xdebug", "-Xrunjdwp:transport=dt_socket,server=y,address=9914,suspend=y", "-Djava.security.egd=file:/dev/./urandom", "-Dspring.profiles.active=dev", "-jar", "/shanoir-ng-datasets.jar"]
+ENTRYPOINT ["/bin/entrypoint", "java", "-Xdebug", "-Xrunjdwp:transport=dt_socket,server=y,address=9914,suspend=y", "-Djava.security.egd=file:/dev/./urandom", "-Dspring.profiles.active=dev", "-jar", "/shanoir-ng-datasets.jar"]
```

Comment:
Let's not debug every time if you agree :)

Reply:
yep!

Reply:
Did you update it ? I don't see the change !

Reply:
I guess it's fine like that : https://github.com/arthursw/shanoir-ng/blob/angular10/docker-compose/datasets/Dockerfile
right?